spec/shadow/index.html

```diff
@@ -1117,6 +1117,20 @@
       </section>
 
       <section>
+        <h3>Focus</h3>
+        <p>A <a>shadow host</a> can delegate focus to its <a>shadow root</a> by assigning a boolean <a href="#widl-ShadowRootInit-delegatesFocus">delegatesFocus</a> flag to be true in <a href="#idl-def-ShadowRootInit">ShadowRootInit</a> dictionary. If omitted, a <a>shadow host</a> does not delegate focus to its shadow root, and the shadow host itself can be focusable.</p>
+        <p>When a shadow host delegates focus, user agent <strong>must</strong> behave as follows.
+          <ol>
+            <li>When the <a>shadow host</a> is being focused by TAB focus navigation: if the direction is forward, the first focusable element in its <a>shadow tree</a> gets focuse. If the direction is backward, the last focusable element in its <a>shadow tree</a> gets focuse. If there is no focusable element in its <a>shadow tree</a>, the <a>shadow host</a> is skipped for the navigation. See the next secition for the formal definition.</li>
+            <li>When the <a>shadow host</a> is being focused by <code>focus()</code> method or <code>autofocus</code> attribute: The first focusable element in its <a>shadow tree</a> gets focus.</li>
```

Comment:
"The first" is no longer good definition here. Without the link, the first usually means  the first element in the tree order.